app/config/WalletTypeConfig.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+import type { WalletType } from '../types/WalletType';
+
+export const WalletTypeOption : {
```

Comment:
What is the rationale having this type twice? (once here and once in `AdaTypesConfig.js`)

Reply:
Yes, Its redundant.
but i kept like for the same reason as wallet object also has two type definition as:

For inside `API`
https://github.com/Emurgo/yoroi-frontend/blob/4d1db2cd4983f5f006c400181d7eafea14414586/app/api/ada/adaTypes.js#L116

For outside `API` 
https://github.com/Emurgo/yoroi-frontend/blob/4d1db2cd4983f5f006c400181d7eafea14414586/app/domain/Wallet.js#L10

I discussed about this before:  
![image](https://user-images.githubusercontent.com/19986226/49362320-db4c6b00-f721-11e8-9eb0-18a571ebdfc2.png)